trove/render/_simple_trovesearch.py

```diff
@@ -9,43 +11,39 @@
 from trove.vocab.jsonapi import JSONAPI_LINK_OBJECT
 from trove.vocab.namespaces import TROVE, RDF
 from ._base import BaseRenderer
-from ._rendering import ProtoRendering, SimpleRendering
+from .rendering import ProtoRendering, SimpleRendering
 if TYPE_CHECKING:
     from trove.util.json import JsonObject
 
+_logger = logging.getLogger(__name__)
+
 
 class SimpleTrovesearchRenderer(BaseRenderer):
     '''for "simple" search api responses (including only result metadata)
 
     (very entangled with trove/trovesearch/trovesearch_gathering.py)
     '''
     PASSIVE_RENDER = False  # knows the properties it cares about
-    _page_links: set[str]
+    INDEXCARD_DERIVER_IRI = TROVE['derive/osfmap_json']  # assumes osfmap_json
+    _page_links: set[str]  # for use *after* iterating cards/card_pages
     __already_iterated_cards = False
 
-    def simple_unicard_rendering(self, card_iri: str, osfmap_json: JsonObject) -> str:
-        raise NotImplementedError
-
-    def simple_multicard_rendering(self, cards: Iterator[tuple[str, JsonObject]]) -> str:
-        raise NotImplementedError
-
-    def unicard_rendering(self, card_iri: str, osfmap_json: JsonObject) -> ProtoRendering:
-        return SimpleRendering(  # type: ignore[return-value]
-            mediatype=self.MEDIATYPE,
-            rendered_content=self.simple_unicard_rendering(card_iri, osfmap_json),
+    def simple_multicard_rendering(self, cards: Iterator[tuple[str, JsonObject]]) -> str | bytes:
+        raise NotImplementedError(
+            f'{self.__class__.__name__} must implement either `multicard_rendering` or `simple_multicard_rendering`'
```

Comment:
The wording of this message is confusing to me. Isn't `multicard_rendering` implemented by default in the next method?